frappe/charts
# first needed modifications to support react

###### Explanation About What Code Achieves:
- It will support a html component as a parent which is needed for react.
- It will store the generated css into a file that it can be used within react

###### Steps To Test:
  - Use branch and create "basic" react component as described in #24.
  - Import and use this component and check if the chart is rendered corretly 

###### TODOs:
  - We should discuss if we want to support React per default -> package.json main should point on a non rollupjs file
